UPGRADE-6.0.md

```diff
@@ -112,6 +112,7 @@ Messenger
  * Use of invalid options in Redis and AMQP connections now throws an error.
  * The signature of method `RetryStrategyInterface::isRetryable()` has been updated to `RetryStrategyInterface::isRetryable(Envelope $message, \Throwable $throwable = null)`.
  * The signature of method `RetryStrategyInterface::getWaitingTime()` has been updated to `RetryStrategyInterface::getWaitingTime(Envelope $message, \Throwable $throwable = null)`.
+ * Deprecated the use of TLS option for Redis Bridge. Replace `redis://127.0.0.1?tls=1` with `rediss://127.0.0.1`.
```

Comment:
For the version `6.0`, I guess it will be removed instead of deprecated, right?

Reply:
You're absolutely correct, it's fixed!

Reply:
It's not fixed actually. Did you forget to push?

Reply:
@nicolas-grekas I was convinced I pushed it, but apparently, I never made that commit (at least there's no trace of it) :neutral_face: 

Should be solved now